rust/examples/tls/Self-Signed mTLS Method/client_certificates.sh

```diff
@@ -33,6 +33,7 @@ openssl req -nodes \
 
 openssl rsa \
           -in end.key \
+          -passin pass:lol \
```

Comment:
why is that needed?

Reply:
I was playing a bit around with certificates and forgot to remove it again